server/server.js

```diff
@@ -56,27 +56,75 @@ app.middleware('parse', bodyParser.urlencoded({
   extended: true
 }));
 
-// The access token is only available after boot
-app.middleware('auth', loopback.token({
-  model: app.models.accessToken,
-  currentUserLiteral: 'me'
-}));
 
-app.use(loopback.token());
+
+// Establish current user from access token
+app.use(loopback.context());
```

Comment:
`loopback.context()` is apparently deprecated, but the alternative is to reimplement the behavior yourself (like digitopia). For now I'm going with it; a lot of examples use it.